code/src/pages/cart/components/CartItem.scss

```diff
@@ -0,0 +1,66 @@
+@import "@design-system";
+
+.spa-cart-item {
+  display: flex;
+  align-items: center;
+  padding: spacing(4) 0;
+  border-bottom: 1px solid var(--spa-cart-item-border-color, $gray-light);
+
+  &__image {
+    width: spacing(25);

```

Comment:
We don't use spacing for width. Just for paddings, margins, gaps etc

Reply:
> We don't use spacing for width. Just for paddings, margins, gaps etc

<img width="369" alt="image" src="https://github.com/idx-academy/spa-orders/assets/56305508/96d342db-8906-4c26-8ff9-ed8332341cac">

you are contradicting yourself.. 
what to use `rem` `em` ?